### DISCUSSION TITLE
I have a idea about using wordwise in Traditional Chinese instead Simplified Chinese

### DISCUSSION THREAD

**Discussion by Jsnnmsc** [xxyzz/WordDumb#246] [General]
Is possible to edit source code of Chinese encoding, and convert to Traditional Chinese from Simplified Chinese?

**Comment by xxyzz** [comment]
No sure what you want to do, Word Wise language can be set to Traditional Chinese in plugin settings.

**Reply by Jsnnmsc** [reply]
Oh really, I'll try later, thanks!
